### ISSUE TITLE
I2S audio device list don't have es9038q2m(or related ),but system seems have sabre driver

### ISSUE THREAD

**Issue by yaoyutian** [moode-player/moode#38]
flac/mp3 music plays normal(select hifiberry dac/or other dac ...),but high bitrate encoder error,seems driver not  functioning!
my board is not Audiophonics,but chip is es9038q2m.
~~even though i add : dtoverlay=i-sabre-q2m to  /boot/config.txt,after reboot ,this txt will recover to hifiberry-dac or other device name in i2s audio device list!!~~(reboot from terminal)
![image](https://user-images.githubusercontent.com/10574966/59779420-cebce800-92ea-11e9-8fe8-61fc350001a5.png)
![image](https://user-images.githubusercontent.com/10574966/59779566-12afed00-92eb-11e9-8dc0-b2c926a9ea74.png)
do i need recompile driver from here? https://github.com/audiophonics/I-Sabre_9038Q2M

**Comment by yaoyutian** [comment]
after reboot from moode and shut down power ,reboot ,/boot/config.txt contents doesn't change;log info seems right,but still some file can't play (mpd error,fail to decode) .
![image](https://user-images.githubusercontent.com/10574966/59782810-59084a80-92f1-11e9-8cc4-0339669f89f7.png)

**Comment by yaoyutian** [comment]
find a modified version support DSD64...

**Comment by moodeaudio** [comment]
moOde I2S driver list only supports devices that use drivers in stock Linux kernel. If your device does not work properly with any of the stock drivers, I cannot help.

**Comment by yaoyutian** [comment]
I found this commit from moode forum:[Audiophonics I-Sabre 9038Q2M DAC driver ](https://github.com/raspberrypi/linux/commit/3d88199c2acde28f6b12d5e9a688a3c003c5e0e0)
And i found  i_sabre_q2m.ko in my burned moode SD system(5.3.1).
So i deemed moode 5.3.1 sould support my hardware play DSD64 .but turns out unable play.
But i searched a lot  forums ,got a modified 5.3.0 moode,burned it ,and it support DSD64.
I also found i_sabre_codec.ko in this system image.  so i guess hardware is ok, may some link error or other,i am trying found out .while my linux skill is at  a low level,i am still learning.
Close!
